### ISSUE TITLE
CNCF Blog for CNSWP survey and cloud native micro-survey

### ISSUE THREAD

**Issue by PushkarJ** [cncf/tag-security#753]
Description: Write a blog post to share results from CNSWP survey and cloud native micro-survey for end user audience

Impact: CNCF PR (Public Relations) team, is already working on this and this issue is intended to be used as a tracker 

Scope: Unclear, but potentially may include reviewing / editing blog post content 

Additional info:
- CNSWP survey results: https://github.com/cncf/surveys/blob/master/security/README.md
- CNSWP Retrospective PR (Pull Request): https://github.com/cncf/tag-security/pull/749
- CNSWP README: https://github.com/cncf/tag-security/tree/main/security-whitepaper
- Micro-survey link: https://www.surveymonkey.co.uk/r/D6QK2ZD

CNSWP: Cloud Native Security White Paper

**Comment by sayantani11** [comment]
Can I work on this? I guess I will be able to do it

**Comment by lumjjb** [comment]
@PushkarJ would you like to (and have bandwidth) to work with or mentor @sayantani11 on this issue?

**Comment by PushkarJ** [comment]
Yes I can help Sayantani. Just to be transparent, most of the blog post will be written by CNCF Public Relations team. We _may_ get a chance to review it and suggest feedback. I will update the issue here if and when I hear back from CNCF PR team. Hope that works :)
